decision-records/0001-schema-improvements.md

```diff
@@ -0,0 +1,114 @@
+# Proposal for Schema Improvements
+
+- Date: 2024-10-22
+
+## Introduction
+
+This RFC proposes improvements to the current data schema to improve usability. Since changing the data structure entails publishing a new major version—which can be costly for downstream dependencies—it is crucial to use this opportunity to reconsider the entire vocabulary and structure.
```

Comment:
```suggestion
This RFC proposes improvements to the current data schema to improve usability. Since changing the data structure entails publishing a new major version, which is costly for downstream dependencies, it is relevant to consider the entire vocabulary and structure to avoid later changes in the near future.
```